app/src/main/kotlin/com/wire/android/mapper/UIParticipantMapper.kt

```diff
@@ -28,7 +30,20 @@ class UIParticipantMapper @Inject constructor(
             membership = userTypeMapper.toMembership(userType),
             connectionState = connectionState,
             unavailable = unavailable,
-            isDeleted = if(user is OtherUser) user.deleted else false
+            isDeleted = if (user is OtherUser) user.deleted else false
+        )
+    }
+
+    fun toUIParticipant(messageReaction: MessageReaction): UIParticipant = with(messageReaction) {
+        return UIParticipant(
+            id = userId,
+            name = name.orEmpty(),
+            handle = handle.orEmpty(),
+            avatarData = previewAsset(wireSessionImageLoader),
+            membership = userTypeMapper.toMembership(userType),
+            unavailable = !deleted && name.orEmpty().isEmpty(),
```

Comment:
Shouldn't it be like this?

```suggestion
            unavailable = deleted && name.isNullOrEmpty()
```

_i.e._ The user is deleted and the name is null or empty?

I'm not sure, could you please double check?